propagation/ogc_processes.md

```diff
@@ -1,33 +1,25 @@
-# OGC API Processes: file-based processing for existing software
+# OGC Application Package - file-based processing for existing software
 
-The OGC API processes specification allows you to expose basically any type of (processing) web service. As a service
-designer, you get the full freedom and responsibility in the definition of your service.
+Another option to create an APEx-compliant service is to package your software using Docker containers. This approach is very generic and is particularly relevant if you have an existing piece of complex software that you want to make available as a service.
+This process adheres to the Best Practices for Earth Observation Application Packaging as defined by the Open Geospatial Consortium (OGC) and the EO Exploitation Platform Common Architecture (EOEPCA) spearheaded by the European Space Agency (ESA). These guidelines ensure that EO applications are reproducible and portable across different execution scenarios.
```

Comment:
General comment: target audience are EO researchers that do not necessarily have an IT background. Lets avoid technical terms (docker, execution scenarios), or make sure to explain it, as the original text was trying to.

'execution scenarios' => should we say 'hosting platforms'? The execution scenarios are abstracted away by a compliant hosting platform anyway, so for the target audience, it perhaps makes more sense to keep common terminology. (Hosting platforms is also what we use in ICG.)